common/appointment/create.ts

```diff
@@ -150,3 +173,14 @@ export const createZoomMeetingForAppointments = async (
         throw new Error(`Zoom - Error while creating zoom meeting: ${e}`);
     }
 };
+
+export const saveZoomMeetingReport = async (appointment: Lecture) => {
+    const meetingReports: Prisma.JsonValue[] = appointment.zoomMeetingReport || [];
```

Comment:
So if two users do this at the same time, we lose one of the meeting reports?

Reply:
Can't we `push` (https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-scalar-lists-arrays#adding-items-to-a-scalar-list)?